App/Components/Announcement.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Image, TouchableOpacity, View, Text, Linking } from 'react-native'
+import ExamplesRegistry from '../Services/ExamplesRegistry'
+import Config from '../Config/AppConfig'
+import DebugConfig from '../Config/DebugConfig'
+import { Images } from '../Themes'
+import styles from './Styles/AnnouncementStyle'
+import RoundedButton from './RoundedButton'
+import { format, isBefore, isAfter } from 'date-fns'
+import { contains, indexOf, last } from 'ramda'
+
+// Example
+ExamplesRegistry.addComponentExample('Announcement', () =>
+  <Announcement
+    style={{height: 350}}
+    // preEvent={}
+    eventDays={[
+      {
+        title: 'Happy Hour',
+        subtitle: 'Come for the fun',
+        eventTimeInfo: 'Stay for the cats',
+        address: '#1 Happy St',
+        buttonUri: 'https://chainreact.squarespace.com',
+        image: {uri: 'https://assets.rbl.ms/4314213/980x.jpg'},
+        buttonText: 'wassaa party'
+      }
+    ]}
+    // postEvent={}
+  />
+)
+
+interface AnnouncementProps {
+  style?: StyleSheet | Object
+  preEvent?: AnnouncementEvent
+  eventDays: Array<AnnouncementEvent>
+  postEvent?: AnnouncementEvent
+}
+
+interface AnnouncementEvent {
+  title: string
+  subtitle: string
+  eventTimeInfo: string
+  address: string
+  image: any
+  buttonUri: string
+  buttonText: string
+}
+
+const correctProps = (props) => {
+  const { hotwireDate } = DebugConfig
+  const { conferenceDates } = Config
+  const { preEvent, eventDays, postEvent } = props
+
+  const today = format(new Date(), 'M/D/YYYY')
+  if (hotwireDate) return eventDays[0]
```

Comment:
you're doing an early return on `hotwireDate` here which would return a string right?